.chloggen/jaegerremotesampling_replace_thrift-gen_with_proto-gen.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
```

Comment:
you can update the changelog to indicate this is an api breaking change.

Reply:
Using this approach it will just add a featuregate for the next releases. I assume it will be removed / a breaking change then in ~0.87.0, right?